doc/Requirements Specification/Parts/Functional Requirements.lyx

```diff
@@ -538,12 +594,12 @@ gls{component}
 \labelwidthstring 00.00.0000
 \begin_inset CommandInset label
 LatexCommand label
-name "/OF120/"
+name "/OF180/"
 
 \end_inset
 
  If requested by the user, Beagle shuts down the computer it's running on
- after it finished.
+ after it finished a measurement.
```

Comment:
What do you want to say? The computer keeps still running during a measurement, while the shut down request is marked?